packages/gatsby-plugin-sitemap/src/internals.js

```diff
@@ -20,29 +20,27 @@ export const defaultOptions = {
           siteUrl
         }
       }
-      allMarkdownRemark(
-        limit: 1000,
+      
+      allSitePage(
+        limit: 1000000000,
```

Comment:
What if I remove limit entirely?

Do you know if that will do the trick or is there a default limit?